reactive_graph/src/effect/immediate.rs

```diff
@@ -214,6 +176,20 @@ mod inner {
         run_done_max: usize,
         /// The [ThreadId] of the run with the highest id.
         /// Used to prevent over-subscribing during parallel execution with the last run.
+        ///
+        /// ```no_run
```

Comment:
Sorry for that! Please use `text` instead as it is not code. With `no_run` attribute, the compiler tries to compile the block.